t/ui/12-needle-edit.t

```diff
@@ -518,6 +518,16 @@ subtest 'Showing new needles limited to the 5 most recent ones' => sub {
       or diag explain \@needle_names;
 };
 
+my $hash  = '{ "name": workaround", "value": "workaround for bsc#123456" }';
+my %tests = (empty => '', string => '"workaround"', hash => $hash);
+foreach my $type (sort keys %tests) {
+    subtest "Needle with $type properties" => sub {
+        $needle_json_file->spurt(qq({ "properties": [ $tests{$type} ] }));
+        $driver->refresh;
+        $driver->title_is('openQA: Needle Editor', 'needle editor shows up');
```

Comment:
IMHO, we should add the check for the workaround values.  E.g. when property has workaround, should check if the workaround checkbox is checked,  the description is correct.

Reply:
Not sure if we need that as we already have other test instructions in this test module that check for property selected, description displayed, etc.

Reply:
IMHO if we do not check if the property's value is shown correctly, how could we know that works well.
I did a test in my local,  the $hash misses one `"` before the workaround, so the needle's json file is not parsed correctly, and the workaround checkox is not checked.